src/RateLimits.sol

```diff
@@ -141,6 +129,8 @@ contract RateLimits is IRateLimits, AccessControl {
 
 }
 
+// TODO: RateLimitHelpers.sol
```

Comment:
I included this because it's very small, but it will probably be modified to include keying on the CCTP domains. I'm good with moving this to a new file, and we should include this `function makeAssetKey(bytes32 key, uint32 cctpDomain) internal pure returns (bytes32)`, but maybe we can do that later if/when it's used.

Reply:
Moved to new file